dcos-log/api/middleware.go

```diff
@@ -63,39 +63,19 @@ func validateToken(t string) (string, error) {
 	if !strings.HasPrefix(t, "token=") {
 		return t, ErrInvalidToken
```

Comment:
This should probably be `ErrMissingToken`, right? What we detect here is the case when no token is present.

`ErrInvalidToken` (and the error message "Invalid token used") must only be used when the token is actually, intrinsically, invalid. Which we do not check here. So it seems like `ErrInvalidToken` can be removed from the code entirely.